bucket/mdcat.json

```diff
@@ -1,23 +1,23 @@
 {
     "version": "2.0.3",
     "description": "cat for markdown",
-    "homepage": "https://github.com/lunaryorn/mdcat",
+    "homepage": "https://github.com/swsnr/mdcat",
     "license": "MPL-2.0",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/lunaryorn/mdcat/releases/download/mdcat-2.0.3/mdcat-2.0.3-x86_64-pc-windows-msvc.zip",
+            "url": "https://github.com/swsnr/mdcat/releases/download/mdcat-2.0.3/mdcat-2.0.3-x86_64-pc-windows-msvc.zip",
             "hash": "c93b31c7b55fe6353059d52c75d84dc7d92546d273c539ed6b5507b5b05fa258"
         }
     },
     "bin": "mdcat.exe",
     "checkver": {
-        "github": "https://github.com/lunaryorn/mdcat",
+        "github": "https://github.com/swsnr/mdcat",
         "regex": "mdcat-([\\d.]+)"
     },
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/lunaryorn/mdcat/releases/download/mdcat-$version/mdcat-$version-x86_64-pc-windows-msvc.zip"
+                "url": "https://github.com/swsnr/mdcat/releases/download/mdcat-$version/mdcat-$version-x86_64-pc-windows-msvc.zip"
             }
         },
         "hash": {
```

Comment:
I wonder it makes sense to remove `hash` section from `autoupdate` completely since BLAKE2 option is [not supported](https://scoop-docs.vercel.app/docs/concepts/App-Manifests.html#optional-properties) for Scoop manifests.

Reply:
I did not notice this, Scoop does not support checksum BLAKE2.